libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryUtils.java

```diff
@@ -21,6 +26,118 @@
   private static final String THREE_STRING_FORMAT = "%s/%s/%s";
   private static final SimpleDateFormat dateFormat = new SimpleDateFormat(DATE_AND_TIME_PATTERN, Locale.US);
   private static final Locale DEFAULT_LOCALE = Locale.US;
+  private static final int UNAVAILABLE_BATTERY_LEVEL = 100;
+  static TelephonyManager telephonyManager = null;
```

Comment:
Why do we need a `static` `TelephonyManager` variable? With a `Context` you can obtain a `TelephonyManager` as was implemented before in `MapEventFactory#obtainCellularNetworkType` 
```java
TelephonyManager manager = (TelephonyManager) context.getSystemService(Context.TELEPHONY_SERVICE);
```
so declaring a `static` variable here won't be necessary.